### ISSUE TITLE
UUID Duplicate Spam

### ISSUE THREAD

**Issue by Barlease** [NillerMedDild/Enigmatica6#2321]
E6 0.4.14

The world for our server was created a few versions ago and upgraded as new versions came out. I think it was created in 0.3.9. In the last few versions, we have had issues with people visiting existing areas (such as someone's base) and the server will start to spam UUID messages about renaming tons and tons of duplicate entities. This continues until the server assumes a crash and restarts itself.

One example here: https://pastebin.com/pPGm85WR
In the example log above, that's only about 1/3rd of the log created from me using a waystone to visit a friend's base.

The images below shows the end of the log where it finally just assumes the server has crashed and restarts it. If this didn't happen, the UUID spam would continue to rename entities.

![spam to crash](https://user-images.githubusercontent.com/82740540/120245958-74499780-c234-11eb-9980-e1667b78265e.JPG)

I just wanted to check if this is a bug, or a world upgrade issue, or if there is an option besides just creating the world from scratch.

**Comment by MuteTiefling** [comment]
From the 0.4.0 changelog:

>It's highly recommended that you make a new world, to get all the new features.
We will not be offering support to 0.4.x worlds generated before 0.4.0. See the bottom of the v0.4.0 changelog for a list of possible issues when not making a new world.

I hate to say it, but a new world is probably in order.

**Comment by zedekblue** [comment]
I'm just dropping a comment for anyone who comes across this, as I was having this issue and managed to solve it to some extent.
It wasn't happening as frequently as OP, however it froze up the server every 5-10 minutes when my mob farm was running. I assumed it was just a lag issue from having a large number of entities to process.

What confused me more it is was always having issues with shulker uuids, despite no shulkers existing in any loaded chunks, which I confirmed via the /debug data.

However I discovered it was simply an issue with the spawners -- for some reason a number of the shulker spawners I had yoinked from end cities very much disliked being converted via the apotheosis method.
One or two would likely cause minimal lag, but almost my entire mob system was using converted shulker spawners.
Changing them out for other types resolved the issue entirely.
